app/src/main/res/values/strings.xml

```diff
@@ -18,49 +18,49 @@
     <string name="address_books_authority_title">Address books</string>
     <string name="help">Help</string>
     <string name="manage_accounts">Manage accounts</string>
-    <string name="please_wait">Please wait …</string>
+    <string name="please_wait">Please wait…</string>
     <string name="send">Send</string>
 
     <string name="notification_channel_crash_reports">Crash Reports</string>
     <string name="notification_channel_debugging">Debugging</string>
     <string name="notification_channel_general">Other important messages</string>
-    <string name="notification_channel_sync">Synchronization</string>
```

Comment:
I just reviewed the strings again and I realised this whole set of titles (`notification_channel_*`) is incorrect.
The place where it's used is the description of the notification channels, not the notifications itself. So "Syncing" for example is wrong, as it should be "Synchronization" the category name. "Syncing stopped. This could be..." is also wrong, as the sync hasn't stopped, it just describes what this notification channel in for. So really, all of this group of changes should be reverted.

Reply:
Fixed(?)